src/test_data/arpes.py

```diff
@@ -363,7 +363,24 @@ def perpendicular_momentum(photon_energy, parallel_momentum,
     $\k_{perpendicular}$ are the surface parallel and perpendicular momentum,
     $E_{ph}$ is the photon energy, $E_{b}$ is the electron binding energy and
     $\Phi$ is the work function.
+    Parameters
+    ----------
+    photon_energy : float or 1D numpy.ndarray,
```

Comment:
These first three can also be integers, can you update to include them? Also we should add to all 3 of these 'if an numpy.array then all three need to have the same length or be of length 1.